python_modules/dagster/setup.py

```diff
@@ -98,6 +98,7 @@ def get_version() -> str:
                 "grpcio-tools==1.32.0",
                 "isort>=4.3.21,<5",
                 "mock==3.0.5",
+                "pandera<0.8.0",
```

Comment:
We should avoid adding a pandera dependency to Dagster itself.

Reply:
If my tests for the docs_snippets require pandera, where should I add it as a dependency?